baml_language/crates/bridge_go/build.sh

```
@@ -0,0 +1,32 @@
+PROTO_DIR="${SCRIPT_DIR}/../bridge_ctypes/types"
+OUT_DIR="${SCRIPT_DIR}/cffi/proto"
+
+mkdir -p "${OUT_DIR}"
+
+# Resolve protoc-gen-go
+if [ -n "${PROTOC_GEN_GO_PATH:-}" ]; then
+    protoc_gen_go_dir="$(dirname "${PROTOC_GEN_GO_PATH}")"
+    export PATH="${protoc_gen_go_dir}:${PATH}"
+elif command -v mise &>/dev/null && mise which protoc-gen-go &>/dev/null; then
+    mise_protoc_path="$(mise which protoc-gen-go)"
+    export PATH="$(dirname "${mise_protoc_path}"):${PATH}"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)